spring-cloud-app-broker-deployer/src/main/java/org/springframework/cloud/appbroker/deployer/AppDeployer.java

```diff
@@ -40,6 +40,10 @@
 		return Mono.empty();
 	}
 
+	/**
```

Comment:
Please, delete the comment to match the other methods.

Reply:
I was trying to make it explicit the non-obvious behavior that the mono might be empty when the update is optimized and skipped, which is currently only visible in the implementation by reading https://github.com/spring-cloud/spring-cloud-app-broker/blob/76765d10775856930874badf430f70e8c3967614/spring-cloud-app-broker-deployer-cloudfoundry/src/main/java/org/springframework/cloud/appbroker/deployer/cloudfoundry/CloudFoundryAppDeployer.java#L1107-L1114
To me this breaks encapsulation between the modules.

What could be a better alternative to this seldom comment ?